sha1/src/lib.rs

```diff
@@ -11,6 +11,18 @@
 //!
 //! # Usage
 //!
+//! ## Simple
+//!
+//! ```rust
+//! use hex_literal::hex;
+//! use sha1::{Sha1, Digest};
+//!
+//! let result = Sha1::digest(b"hello world");
+//! assert_eq!(result[..], hex!("2aae6c35c94fcfb415dbe95f408b9ce91ee846ed"));
+//! ```
+//!
+//! ## Verbose
```

Comment:
Maybe this should be something like "Incremental" or "Initialize-Update-Finalize API" /cc @newpavlov

Reply:
I agree. I also think it's better to use something like "One-shot API" instead of "Basic".

Reply:
> I also think it's better to use something like "One-shot API" instead of "Basic".

You mean replacing "Simple" with "One-shot API"?